cmd/atlante/cmd/root.go

```diff
@@ -131,10 +146,28 @@ func rootCmdRun(cmd *cobra.Command, args []string) error {
 			ExitCode:  1,
 		}
 	}
-	ctx, cancel := context.WithCancel(context.Background())
+	if timeout != 0 {
```

Comment:
Remove the var declarations at 133, drop the else statement at 153 and use line 154 before the if on 149